DataMigration/src/main/java/org/sagebionetworks/dian/datamigration/DataMigration.java

```diff
@@ -103,6 +103,16 @@ private static void runDataMigration() throws SynapseException, IOException {
 
         System.out.println("Completed data migration successfully");
     }
+
+    private static void runManualAccountMigration(String[] args) throws IOException {
```

Comment:
I don't see where `runManualAccountMigration` is called.

Reply:
It isn't called anywhere.  Should I mention that in the function header?  I'm just adding this so that the JAR tool I exported can exist in the code base.

Reply:
I may be missing something but I don't see how `runManualAccountMigration` can be called from an executable .jar. Such an executable has as its entry point the `public static main()` function of some public class.

Reply:
I can see the confusion. When I made the JAR, I replaced the contents of public static main() with runManualAccountMigration() to call the function. This PR just adds the building blocks to create a JAR, in case someone else wants to make it in the future or it needs re-created.

Gradle only seems to support creating a JAR off of one file, which in this project's case is DataMigration.java.  Otherwise, I would put this in it's own file. With that in mind, do you have a suggestion for a different architectural approach?

Reply:
> When I made the JAR, I replaced the contents of public static main() with runManualAccountMigration() to call the function.

It sounds like you are saying that you modify the source code at the time you build the .jar file.

> Gradle only seems to support creating a JAR off of one file, which in this project's case is DataMigration.java. 

I have not used Gradle but it sounds like its behavior follows that of Java:  When you build a .jar file you can add a manifest that says which public class is the entry point. (See https://docs.oracle.com/javase/tutorial/deployment/jar/appman.html). This does not mean you can have only one entry point but rather that there is one that is used by default when you run `java -jar my_jar_file.jar`.  You can override the entry point by specifying it at run time, e.g., see part 5 of this tutorial: https://www.baeldung.com/java-run-jar-with-arguments

> With that in mind, do you have a suggestion for a different architectural approach?

Yes, I would add a public class `ManualMigration` and make `runManualAccountMigration` a public, static method of that class.  I would add to the README a section on manual migration with instructions on how to execute (something like `java -cp the_jar_file.jar org.sagebionetworks.dian.datamigration.ManualMigration`